diff --git a/config.h b/config.h
index 41e2b8f..0987dba 100644
--- a/config.h
+++ b/config.h
@@ -2,12 +2,12 @@
 
 /* appearance */
 static const char font[]        = "monospace:size=9";
-static const char* normbgcolor  = "#222222";
-static const char* normfgcolor  = "#cccccc";
-static const char* selbgcolor   = "#555555";
-static const char* selfgcolor   = "#ffffff";
-static const char* urgbgcolor   = "#111111";
-static const char* urgfgcolor   = "#cc0000";
+static const char* normbgcolor  = "#282a36";
+static const char* normfgcolor  = "#6272a4";
+static const char* selbgcolor   = "#44475a";
+static const char* selfgcolor   = "#f8f8f2";
+static const char* urgbgcolor   = "#ff5555";
+static const char* urgfgcolor   = "#f8f8f2";
 static const char before[]      = "<";
 static const char after[]       = ">";
 static const char titletrim[]   = "...";
